docs/Developer Documentation/ResumeController/index.md

```diff
@@ -0,0 +1,224 @@
+# Resume Controller
+
+This page will descrive internal structure and detailed design of Resume controller 
+
+|||
+Figure 1: ResumeController Overview
+![ResumeControllerClasses](./assets/resume_controller_classes.png)
+|||
+
+!!! NOTE
+Classes named like *Impl only _represent_ implementations of the abstract sub classes and may not be named the same in the SDL Core project.
+UML Refresher
```

Comment:
@LuxoftAKutsan please add a new line because it looks like 

> !!! NOTE Classes named like *Impl only represent implementations of the abstract sub classes and may not be named the same in the SDL Core project. UML Refresher

Reply:
Literally it is not a markdown. This will be generated to HTML page. Same as guidelines. Take an example of https://github.com/LuxoftSDL/sdl_core_guides/blob/master/docs/Developer Documentation/Transport Manager/index.md and https://smartdevicelink.com/en/guides/core/developer-documentation/transport-manager/